fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/stores/HashJobStore.java

```diff
@@ -51,4 +61,43 @@ public HashJobStore() {
     public Observable<JobRow> getAllRows() {
         return Observable.from(store.values());
     }
+
+    @Override
+    public Observable<JobRow> getFilteredRows(Set<ApiJobStoreFilter> apiJobStoreFilters)
+            throws IllegalArgumentException {
+        return Observable.from(
+                store.entrySet().stream()
```

Comment:
Since we don't care about the keys, we can just stream over the `store.values()`. Then we don't have to worry about extracting the value from the entries.